algokit-client-generator-ts/package.json

```diff
@@ -6,7 +6,7 @@
   "scripts": {
     "build": "run-s build:*",
     "build:0-clean": "rimraf dist",
-    "build:1-schema-to-types": "cd src/schema && npx --yes json-schema-to-typescript application.schema.json > application.d.ts",
+    "build:1-schema-to-types": "cd src/schema && npx --yes json-schema-to-typescript application.json > application.d.ts",
```

Comment:
@tristanmenzel im confused by this change?

Reply:
accidental - was trying to workout a weird issue with the schema not building but it turned out to be a windows path bug